components/custom_status/custom_status_modal.tsx

```diff
@@ -136,7 +127,7 @@ const CustomStatusModal: React.FC<Props> = (props: Props) => {
         let rightOffset = Constants.DEFAULT_EMOJI_PICKER_RIGHT_OFFSET;
         const target = getCustomStatusControlRef();
         if (target) {
-            const anyTarget: any = target;
+            const anyTarget: Element = target;
```

Comment:
Variable name should be changed from anyTarget to something else.

Reply:
Will update as discussed